garage/envs/mujoco/sawyer/pick_and_place_env.py

```diff
@@ -0,0 +1,114 @@
+from gym.envs.robotics import rotations
+from gym.spaces import Box
+import numpy as np
+
+from garage.core.serializable import Serializable
+from garage.envs.base import Step
+from garage.envs.mujoco.mujoco_env import MujocoEnv
+from garage.misc.overrides import overrides
+
+
+def goal_distance(goal_a, goal_b):
```

Comment:
please make this private `def _goal_distance(goal_a, goal_b)` or make it a static method 

```python
class PickAndPlaceEnv():

    @staticmethod
    def _goal_distance(goal_a, goal_b):
        # stuff
```

Reply:
Done.